src/types/types.ts

```diff
@@ -713,15 +718,28 @@ export type DeleteAssetMapping3D = AssetMapping3DBase;
 export type EXECUTE = 'EXECUTE';
 
 /**
- * Query schema for asset aggregate endpoint
+ * Query schema for event aggregate endpoint
  */
-export interface EventAggregateQuery {
+export type EventAggregateQuery = EventCountAggregate | EventAdvancedAggregate;
+
+export interface EventCountAggregate {
   /**
    * Filter on events with strict matching.
    */
   filter?: EventFilter;
 }
 
+export interface EventAdvancedAggregate extends EventCountAggregate {
+  /**
+   * The field name(s) to apply the aggregation on. Currently limited to one field.
+   */
+  fields: ('type' | 'subtype' | 'dataSetId')[];
+  /**
+   * Type of aggregation to apply. values - Get unique values (upto max 1000) in the specified field ordered by frequency.
+   */
+  aggregate: 'values';
```

Comment:
Did you consider `uniqueValues` instead of `values`? Think it's more explicit

Reply:
The schema is defined here: https://docs.google.com/document/d/1xrts3tu_7ldJDNiY6RJQ4Qt48onSC5R-8pmtiM9MViA/edit#bookmark=id.5x5u3i4p67l4

Reply:
`values` seems a bit generic though. I first reviewed the test and just looking at the code, I couldn't understand what the result of the request was